### PR TITLE
batches: replace video on index and leave notes on md files

### LINKED ISSUE #42412: the batch changes demo video is very outdated
I was clicking through the product feature pages on sourcegraph.com and I noticed the video on https://sourcegraph.com/batch-changes is very old. Sourcegraph looks and behaves completely differently now, as well as Batch Changes. We may want to consider content that is more time durable, or a new demo video on this page.

cc @malomarrec

### PR DESCRIPTION
**Pull request by adeola-ak**
closes #42412
update the batch changes demo video to include a flow for ssbc on the batch change index page as well as our docs

https://user-images.githubusercontent.com/67931373/196997252-487d845f-7b58-47a6-bd46-2061edaaf307.mp4

## Test plan
tested views of the video

## App preview:

- [Web](https://sg-web-aa-demo-video.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.

### CODE REVIEW

**Comment by sg-e2e-regression-test-bob** [comment]
## Bundle size report 📦

| Initial size | Total size | Async size | Modules |
| --- | --- | --- | --- |
| 0.00% (0.00 kb) | 0.00% (+0.40 kb) | 0.00% (+0.40 kb) | 0.00% (0) |

Look at the [Statoscope report](https://console.cloud.google.com/storage/browser/_details/sourcegraph_reports/statoscope-reports/aa/demo-video/compare-report.html;tab=live_object) for a full comparison between the commits cfe55db and c6bc072 or [learn more](https://docs.sourcegraph.com/dev/how-to/testing#bundlesize).


<details>
<summary>Open explanation</summary>

- `Initial size` is the size of the initial bundle (the one that is loaded when you open the page)
- `Total size` is the size of the initial bundle + all the async loaded chunks
- `Async size` is the size of all the async loaded chunks
- `Modules` is the number of modules in the initial bundle
</details>

**Comment by adeola-ak** [comment]
If we like the video ill update the docs where i left the to-do comments

**Review by chrispine — APPROVED** [review]
LGTM, though let's wait for a few more folks on the team to get a chance to watch it

Thanks!

File: doc/batch_changes/index.md [hunk 1]
```diff
@@ -47,6 +47,7 @@ Create a batch change by specifying a search query to get a list of repositories
 
 ## Getting started
 
+//to-do
```
Comment (courier-new) [inline doc/batch_changes/index.md:50]: Was the intent of this to update the YouTube link in this guide? I think that's a great idea if so. Could we talk to #it-tech-ops to get your video posted on YouTube?

File: doc/dev/background-information/batch_changes/index.md [hunk 2]
```diff
@@ -7,6 +7,7 @@ Welcome, new batch change developer! This section will give you a rough overview
 > NOTE: Never hesitate to ask in [`#batch-changes-internal`](https://sourcegraph.slack.com/archives/C01LJ9DK8ES) for help!
 
 ### What are batch changes?
+//to-do
```
Comment (courier-new) [inline doc/dev/background-information/batch_changes/index.md:10]: (Guessing the same is meant for this?)

File: client/web/src/enterprise/batches/list/GettingStarted.tsx [hunk 3]
```diff
@@ -170,16 +170,23 @@ export const GettingStarted: React.FunctionComponent<React.PropsWithChildren<Get
...
-                className="percy-hide chromatic-ignore"
-                width="100%"
-                height="600"
-                src="https://www.youtube-nocookie.com/embed/eOmiyXIWTCw"
-                frameBorder="0"
-                allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture"
-                allowFullScreen={true}
-            />
+            <video
+                className="w-100 h-auto shadow percy-hide"
```
Comment (courier-new) [inline client/web/src/enterprise/batches/list/GettingStarted.tsx:174]: Should we put `chromatic-ignore` on this for good measure, too?

**Review by courier-new — APPROVED** [review]
I love this, well done! And I can't believe you got that all in one continuous take, too! Or if it was multiple takes, it was so smooth I didn't notice haha. 🎉 Approving to unblock, but had 2 small questions about the code changes. 😄

**Review by Piszmog — APPROVED** [review]

**Pull request merged**